tests/test_strategy_infinite_bus.py

```diff
@@ -333,3 +333,66 @@ def testing_event_market_cycle_posting_bids(bus_test4, area_test4):
     assert bus_test4._bids[area_test4.test_market.id][-1].energy == sys.maxsize
     assert bus_test4._bids[area_test4.test_market.id][-1].price == 25 * sys.maxsize
     ConstSettings.IAASettings.MARKET_TYPE = 1
+
+
+"""TEST5"""
+
+
+@pytest.fixture()
+def area_test5():
+    return FakeArea(0)
+
+
+@pytest.fixture()
+def bus_test5(area_test5):
+    c = InfiniteBusStrategy(energy_rate_profile="src/d3a/resources/SAM_SF_Summer.csv")
+    c.area = area_test5
+    c.owner = area_test5
+    return c
+
+
+def test_global_market_maker_rate_profile(bus_test5, area_test5):
+    assert isinstance(GlobalConfig.market_maker_rate, dict)
```

Comment:
Sorry, I just noticed this part. What is this test doing? Why does it need fixtures, and what does it have to do with the Infinite Bus strategy?

Reply:
> Why does it need fixtures, and what does it have to do with the Infinite Bus strategy?

InfiniteBus strategy also sets the global market maker rate that is used by other strategies as a reference.